src/actions.cpp

```diff
@@ -625,8 +625,8 @@ namespace Action {
             std::ostringstream os;
             // #1114 - show negative values for SByte
             if (md.typeId() == Exiv2::signedByte) {
-                for ( int c = 0 ; c < md.value().count() ; c++ ) {
-                    int value = md.value().toLong(c);
+                for ( long c = 0 ; c < md.value().count() ; c++ ) {
+                    long value = md.value().toLong(c);
```

Comment:
The return type of [`md.value().count()`](https://github.com/Exiv2/exiv2/blob/fec1813a3e7b0e4aaed7fae7ba9f9b49ca526638/include/exiv2/value.hpp#L123) is `long`. Also, the parameter of [`md.value().toLong()`](https://github.com/Exiv2/exiv2/blob/fec1813a3e7b0e4aaed7fae7ba9f9b49ca526638/include/exiv2/value.hpp#L152) is `long`. So `long` is a better choice for the type of `c`.